src/vhost_user/slave_req_handler.rs

```diff
@@ -800,6 +764,15 @@ impl<S: VhostUserSlaveReqHandler> SlaveReqHandler<S> {
     }
 }
 
+/// Utility function to take the first element from option of a vector of files.
```

Comment:
nit: Should also mention here that this will return `None` if there is more than one file in the Vec.

Reply:
Added a comment